contracts/governance/utils/VotesConfidential.sol

```diff
@@ -63,6 +63,14 @@ abstract contract VotesConfidential is Nonces, EIP712, IERC6372 {
         return _delegateCheckpoints[account].latest();
     }
```

Comment:
Yes, but why _not_ introducing new public functions & instead updating existing functions?
```
    /**
     * @dev Returns the current amount of votes that `account` has.
     * A transaction is required to get access to.
     */
    function getVotes(address account) public virtual returns (euint64 votes) {
        _validateVotesAccess(account);
        votes = _delegateCheckpoints[account].latest();
        FHE.allow(votes, account);
    }
```

Reply:
We still need to expose view functions for offchain infra

Reply:
Why not having offchain clients calling `getVotes(..)` statically?